index.bs

```diff
@@ -857,6 +857,12 @@ sequence&lt;{{WebTransportHash}}&gt; |serverCertificateHashes|, run these steps.
     1. Let |error| be a newly [=DOMException/created=] {{WebTransportError}} whose
        {{WebTransportErrorOptions/source}} is `"session"`.
     1. [=Cleanup=] |transport| with |error|.
+
+       Note: Redirects are not followed. Network errors caused by redirection are intentionally
+       indistinguishable from other network errors.  In cross-origin contexts, this would reveal
+       information that would normally be blocked by CORS. In same-origin contexts, it would
+       encourage users to abuse the handshake as a vector for passing information.
```

Comment:
"Users" seems ambiguous and tends to refer to end-users. How about:
```suggestion
       information that would normally be blocked by CORS. In same-origin contexts, it might
       encourage applications to abuse the handshake as a vector for passing information.
```